telegram/src/main/kotlin/com/github/kotlintelegrambot/dispatcher/HandlersDsl.kt

```diff
@@ -165,3 +167,11 @@ fun Dispatcher.dice(body: HandleDice) {
 infix fun Handler.requires(predicate: (Update) -> Boolean): Handler {
     return ExtendedHandler(this, predicate)
 }
+
+fun <T> Handler.inputChain(
+    filter: Filter = Filter.Text,
+    inputChainIdGen: (previousInputChainId: T?, Update) -> T?,
+    handleInputChain: HandleInputChain<T>,
+): Handler {
+    return InputChainHandler(filter, this, inputChainIdGen, handleInputChain)
+}
```

Comment:
It would be good to also have an example usage of this in the doc and/or one of the sample modules.